yo/db.py

```diff
@@ -73,29 +73,19 @@
      mysql_engine='InnoDB',
 )
 
-# We basically just store one entry for each configured transport, and delete them with API calls if required
-user_transports_table = sa.Table('yo_user_transports', metadata,
+# TODO: at some point turn this back into a normalised SQL table and/or add logging so users can see when their settings were changed etc
+user_settings_table = sa.Table('yo_user_settings', metadata,
      sa.Column('tid', sa.Integer, primary_key=True),
-
      sa.Column('username', sa.String(20), index=True),
-
-     sa.Column('notify_type', sa.String(10), nullable=False, index=True),
-     sa.Column('transport_type', sa.String(10), nullable=False, index=True),
-
-     sa.Column('sub_data', sa.String(1024), index=False),
-
-     sa.Column('created_at', sa.DateTime, default=sa.func.now(), index=True,
-               doc='Datetime when we first created this user preferences entry'),
-     sa.Column('updated_at', sa.DateTime, default=sa.func.now(), index=True,
-               doc='Datetime when preferences were updated'),
+     sa.Column('transports', sa.UnicodeText, index=False,
```

Comment:
It's so that we don't need to rebuild the JSON object every time get_transports is called - makes the SQL schema less complex than it was previously and means that we only need to update a single row when changing transports, making it more atomic and meaning we can migrate to databases that don't support transactions with greater ease.

That said, i'm all for going back to a normalised schema - but not while we're on a tight deadline.